packages/salesforcedx-sobjects-faux-generator/src/types/index.ts

```diff
@@ -8,9 +8,40 @@
 export {
   BatchRequest,
   BatchResponse,
-  SObject,
+  ChildRelationship,
   Field,
-  ChildRelationship
+  SObject
 } from './describe';
-
 export { SObjectCategory, SObjectRefreshSource } from './general';
+export { SObjectDefinition };
+
+import { SObjectDefinition } from '../generator/types';
```

Comment:
We should probably move `SObjectDefinition` to this module instead of keeping it in `generator`.

Reply:
@lcampos Will do. I'll take a closer look at some of the other definitions as well.